complex_functions/strassen.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
```

Comment:
Same as your last PR, you can add the comment stating that this function is not ours